conn.go

```diff
@@ -78,7 +83,7 @@ func Server(conn *ipv4.PacketConn, config *Config) (*Conn, error) {
 
 	c := &Conn{
 		queryInterval: defaultQueryInterval,
-		queries:       map[string]chan queryResult{},
+		queries:       []query{},
```

Comment:
Would we be able to use a `map[string][]query` here, so that we can retain the ability to go straight to the list of queries with that name? Or is that overkill in this case?

Reply:
WebRTC uses the same id for every interface, so it won't save us anything for our use cases unfortunately. 

We will just end up with a map with one key, and then everything in that slice.